### DISCUSSION TITLE
Minimum and Recommended System Requirements

### DISCUSSION THREAD

**Discussion by HaidarZ** [saleor/saleor#6889] [Deployment]
Hi guys,

I'm wondering if there's any written or agreed on minimum and recommended system requirements.
It would be helpful to check the hosting costs and other stuff.

Thanks !

**Comment by Reef3rm4n** [comment]
Upon start the app eats up around 400mb of memory so i would suggest atleast 800mb of ram and 2v cores.

**Comment by krzysztofwolski** [comment]
I would also suggest using "more instances" than "bigger instances", which is helpful for granular load balancing (and in case of one instance failure there are other ones to response).
